React/Views/RCTNavigator.m

```diff
@@ -332,6 +345,15 @@ - (void)setPaused:(BOOL)paused
   }
 }
 
+- (void)setInteractivePopGestureEnabled:(BOOL)interactivePopGestureEnabled {
```

Comment:
nit `{` should go on a newline and method contents should be indented with 2 spaces